### DISCUSSION TITLE
Paging breakouts or pagination in the editor

### DISCUSSION THREAD

**Discussion by shanthi-techie** [react-page/react-page#1122] [Q&A]
How to implement paging breakouts/page numbers/pagination in the editor so that the experience would be similar to using MS word. If there are page numbers allowed user can restrict the content to should go to each page.

**Comment by macrozone** [comment]
@shanthi-techie the editor focuses on creating web page content, not paper content or pdf content. (despite the name...).

if you want to print, there are css properties for forced page-breaks that you could use.

**Reply by shanthi-techie** [reply]
Thank you for checking this
